telegram/ext/filters.py

```diff
@@ -961,7 +970,25 @@ def filter(self, message: Message) -> bool:
             return message.chat.type in [Chat.GROUP, Chat.SUPERGROUP]
 
     group = _Group()
-    """Messages sent in a group chat."""
+    """Messages sent in a group or a supergroup chat."""
+
+    class _SuperGroup(MessageFilter):
+        name = 'Filters.supergroup'
+
+        def filter(self, message: Message) -> bool:
+            return message.chat.type == Chat.SUPERGROUP
+
+    supergroup = _SuperGroup()
+    """Messages sent in a supergroup chat."""
+
+    class _ChatTypes(_ChatType):
+        """Filter using chat type"""
```

Comment:
This class is private, it shouldn't get a docstring

Reply:
yes, sure. will replace this to line:992